extension/src/test/suite/experiments/index.test.ts

```diff
@@ -878,6 +878,29 @@ suite('Experiments Test Suite', () => {
       )
     }).timeout(WEBVIEW_TEST_TIMEOUT)
 
+    it('should be able to handle a message to update the table depth', async () => {
+      const { experiments } = buildExperiments(disposable, expShowFixture)
+
+      const webview = await experiments.showWebview()
+
+      const mockMessageReceived = getMessageReceivedEmitter(webview)
+      const executeCommandStub = stub(commands, 'executeCommand')
+
+      const messageReceived = new Promise(resolve =>
+        disposable.track(mockMessageReceived.event(() => resolve(undefined)))
+      )
+
+      mockMessageReceived.fire({
+        type: MessageFromWebviewType.SET_EXPERIMENTS_HEADER_DEPTH
+      })
+
+      expect(executeCommandStub).to.be.calledWith(
```

Comment:
While creating this test, I realized that we don't have a telemetry event set up for this command. Will add in a followup!

Reply:
Seems like that test is actually failing

Reply:
Figured it out! I now realize I have completely misunderstood how the end to end tests work and need to rewrite this. Converting to a draft for now!